src/CovidData/Context.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {
+  createContext,
+  FunctionComponent,
+  useState,
+  useEffect,
+  useCallback,
+  useContext,
+} from "react"
+import { useConfigurationContext } from "../ConfigurationContext"
+
+import * as CovidData from "./covidData"
+import { fetchCovidDataForState } from "./covidDataAPI"
+
+export type RequestStatus = "SUCCESS" | "LOADING" | "ERROR" | "MISSING_INFO"
+
+export type CovidDataRequest = {
+  status: RequestStatus
+  data: CovidData.CovidData
+  error?: string
+}
+
+export type CovidDataContextState = {
+  locationName: string
+  request: CovidDataRequest
+}
+
+const initialRequest: CovidDataRequest = {
+  status: "MISSING_INFO",
+  data: [],
+}
+
+const initialState: CovidDataContextState = {
+  locationName: "",
+  request: initialRequest,
+}
+
+export const CovidDataContext = createContext<CovidDataContextState>(
+  initialState,
+)
+
+export const CovidDataContextProvider: FunctionComponent = ({ children }) => {
+  const { stateAbbreviation, displayCovidData } = useConfigurationContext()
+  const [request, setCovidDataRequest] = useState(initialRequest)
```

Comment:
Is there a reason this doesn't use the conventional naming scheme?

Reply:
nope, can fix!